pyhoma/client.py

```diff
@@ -123,40 +130,49 @@ async def register_event_listener(self) -> str:
 
         return listener_id
 
-    async def fetch_event_listener(self, listener_id: str) -> List[Event]:
+    async def fetch_events(self) -> List[Event]:
         """
         Fetch new events from a registered event listener. Fetched events are removed
         from the listener buffer. Return an empty response if no event is available.
         Per-session rate-limit : 1 calls per 1 SECONDS period for this particular
         operation (polling)
         """
-        response = await self.__post(f"events/{listener_id}/fetch")
+        response = await self.__post(f"events/{self.event_listener_id}/fetch")
         events = [Event(**e) for e in humps.decamelize(response)]
 
         return events
 
-    async def unregister_event_listener(self, listener_id: str) -> None:
+    async def unregister_event_listener(self) -> None:
```

Comment:
Should `unregister_event_listener` and `register_event_listener` not use backoff?

Reply:
If you are logout, you don't need to unregister the listener as this one has been deleted. 
About register, normally you do it just after a login, it's why I didn't add a retry logic.

Reply:
Makes sense, let's see how it works like this. We can always add it if we face issues.